docs/operate-and-deploy/high-availability.md

```diff
@@ -0,0 +1,16 @@
+# High availability
+
+When you run pull queries, it’s often the case that you need your data to remain available for querying even if the server fails. Because ksqlDB supports clustering, it can remain highly available to support pull queries on replicas of your data, even in the face of partial cluster failure.
```

Comment:
Suggest line breaks at col 80.

Reply:
```suggestion
When you run pull queries, it’s often the case that you need your data to remain available for querying even if one server fails. Because ksqlDB supports clustering, it can remain highly available to support pull queries on replicas of your data, even in the face of partial cluster failures.
```